media/js/firefox/new/experiment-firefox-new-batm-anim.js

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var cop = new Mozilla.TrafficCop({
+        id: 'experiment_firefox_new_batm_anim',
+        variations: {
+            'xv=batmprivate': 33,   // Control: basic page, no machine
```

Comment:
The only solution I can think of that wouldn't require changes to traffic cop would be to show these new template variations using a `v` parameter instead, so `?xv=batmprivate&v=batma` etc.